jetty-server/src/test/java/org/eclipse/jetty/server/StressTest.java

```diff
@@ -458,24 +458,19 @@ private static class TestHandler extends HandlerWrapper
         @Override
         public void handle(String target, final Request baseRequest, final HttpServletRequest request, final HttpServletResponse response) throws IOException, ServletException
         {
-            long now = TimeUnit.NANOSECONDS.toMillis(System.nanoTime());
+            long now = NanoTime.now();
             long start = Long.parseLong(baseRequest.getHeader("start"));
-            long received = baseRequest.getTimeStamp();
 
             _handled.incrementAndGet();
-            long delay = received - start;
-            if (delay < 0)
-                delay = 0;
+            long delay = NanoTime.millisElapsed(start, now);
             _latencies[2].add(delay);
-            _latencies[3].add((now - start));
+            _latencies[3].add(delay);
```

Comment:
I don't think this is the same logic?

Reply:
It's not exactly the same logic, but before it was a garbage value, since `baseRequest.getTimeStamp()` returns an epoch millis time, and `delay` was calculated mixing up epoch millis with current nanoTime converted to millis -- a bogus number that could have even be negative.

Maybe `baseRequest.getTimeStamp()` should return a nanoTime, but `CustomRequestLog` relies on it to be a UTC dateTime -- feels like we should either use `Instant` or have 2 values (the nanoTime and the dateTime).

Reply:
Ok. It's only a test anyway, so if it is a nonsense value, perhaps delete it.